ilastik/applets/tracking/base/trackingBaseDataExportApplet.py

```diff
@@ -19,30 +19,37 @@
 # on the ilastik web site at:
 #		   http://ilastik.org/license.html
 ###############################################################################
+from ilastik.applets.base.appletSerializer import SerialSlot, SerialDictSlot
 from ilastik.applets.dataExport.dataExportApplet import DataExportApplet
+from ilastik.applets.dataExport.dataExportSerializer import DataExportSerializer
 from ilastik.applets.tracking.base.opTrackingBaseDataExport import OpTrackingBaseDataExport
 from ilastik.utility import OpMultiLaneWrapper
 import os
 
 class TrackingBaseDataExportApplet( DataExportApplet ):
     """
     This a specialization of the generic data export applet that
-    provides a special viewer for trackign output.
+    provides a special viewer for tracking output.
     """
-    def __init__(self, workflow, *args, **kwargs):
-        if 'default_export_filename' in kwargs:
-            default_export_filename = kwargs['default_export_filename']
-            del kwargs['default_export_filename']
-        else:
-            default_export_filename = ""
-
+    def __init__(self, workflow, title, default_export_filename=''):
```

Comment:
shouldn't this at leas take the `is_batch` argument (in case of batch processing?), see https://github.com/ilastik/ilastik/blob/126d685311aca8e4f2b82462c4d9109c64d97149/ilastik/applets/dataExport/dataExportApplet.py#L34-L40

Reply:
:white_check_mark: